src/contrib/mlir/core/compiler.cpp

```diff
@@ -60,6 +60,10 @@
 // Defines a new LLVM debug type for this file to be used by LLVM_DEBUG macro.
 #define DEBUG_TYPE "mlir-compiler"
 
+static llvm::cl::opt<bool> clEnableNgOpFusion("ngraph-op-fusion",
+                                              llvm::cl::init(false),
+                                              llvm::cl::desc("Enable ngraph op fusion pass"));
```

Comment:
Please, update this comment to be more specific and update flag variable accordingly.